src/components/Modal/Modal.module.scss

```diff
@@ -0,0 +1,107 @@
+@import '~@palmetto/palmetto-design-tokens/build/scss/variables-size';
+
+@keyframes fadeIn {
+  0% {
+    opacity: 0;
+  }
+
+  100% {
+    opacity: 1;
+  }
+}
+
+@keyframes zoomIn {
+  from {
+    transform: scale3d(0.3, 0.3, 0.3);
+    opacity: 0;
+  }
+
+  50% {
+    opacity: 1;
+  }
+}
+
+.modal-close {
+  border: none;
+  background: transparent;
+  padding: var(--modal-close-button-padding);
+  color: var(--modal-close-button-color);
+
+  &:hover {
+    color: var(--modal-close-button-color-hover);
+  }
+}
+
+:global([data-reach-dialog-overlay]) {
+  display: flex;
+  position: fixed;
+  top: 0;
+  right: 0;
+  bottom: 0;
+  left: 0;
+  align-items: flex-end;
+  z-index: var(--size-z-index-overlay);
+  background: hsla(0, 0%, 0%, 0.33);
```

Comment:
Might be worth getting some tokens that represent semi-transparent colors. Not for this PR.